src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -120,7 +127,33 @@ const CityColumnChart = (props) => {
       enabled: false,
     },
     tooltip: {
-      enabled: false,
+      backgroundColor: '#fff',
+      borderColor: colorDark,
+      borderRadius: 4,
+      // shared puts multiple series in the same tooltip
+      // but also means you can hover anywhere on the chart to get the tooltip
+      // (even for single series)
+      shared: true,
+      useHTML: true,
+      headerFormat: '<div class="cityColumnChartTooltipHeader">{point.key}</div>',
+      pointFormatter() {
+        const formattedValue = numeral(this.y).format(firstIndicator.format);
```

Comment:
Seems reasonable to assume we will never have indicators with different formats.